tests/test_multiplication.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
Didn't get these tests. You implemented __mul__ and __rmul__ magic methods, so need to verify multiplication Pixel by number, number by Pixel